ci-services/drone.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const env = process.env
+
+/**
+ * Should update the `package-lock.json`
+ */
+function shouldUpdate () {
+  let re = /^(chore|fix)\(package\): update [^ ]+ to version.*$/mi
```

Comment:
Like Codeship there isn't a good way to get back the current number of commits on the branch so here we're matching the commit message. Might be a good feature request for @bradrydzewski at some point.